eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/VariantSourceEntityRepository.java

```diff
@@ -36,4 +39,14 @@
 
     List<VariantSourceEntity> findByStudyIdOrStudyName(String studyId, String studyName);
 
+    List<VariantSourceEntity> findByStudyIdIn(List<String> studyIds, Pageable pageable);
+
+    long countByStudyIdIn(List<String> studyIds);
+
+    @Query(value = "{fid: {$in : ?0}}}", fields = "{fid : 1, samp : 1}")
```

Comment:
The reason I'm using this is because at the moment the only client code that uses this query method just requires the file ID and the sample name, thus I'm using a projection to only return these values.  
Do you think it would be better to remove the projection, in case future client code needs more of the document returning, to save it needing to be removed then?